frontend/server/src/main/java/com/amazonaws/ml/mms/http/ManagementRequestHandler.java

```diff
@@ -132,6 +132,8 @@ private void handleDescribeModel(ChannelHandlerContext ctx, String modelName)
         resp.setMaxBatchDelay(model.getMaxBatchDelay());
         resp.setMaxWorkers(model.getMaxWorkers());
         resp.setMinWorkers(model.getMinWorkers());
+        resp.setModelInitialization(
```

Comment:
the response is a json, the value is expected to be parsed by program.
It's better use boolean type.
The human readable string can cause localization issue in future.
And sole to key name is part of API. It's better name it as something like "StartupModel"

Reply:
Aah, true. I can use that as well. I thought I could be more descriptive, but we don't need to do it. Probably uses less space as well.